Sprint-1/errors/1.js

```diff
@@ -2,3 +2,6 @@
 
 const age = 33;
 age = age + 1;
+console.log(age)
+//TypeError: Assignment to constant variable.
+    //he const keyword is initialized once,can not change or update the constant variable twice
```

Comment:
Your interpretation is correct, but you also need to fix the error too.
Based on your understanding, how would you edit the code to make it run properly?